Analysis/output/country_data_report.Rmd

```diff
@@ -18,14 +25,14 @@ library(taxdat)
 library(sf)
 ```
 
-# Mapping results summary for `r stringr::str_extract(params$config, "(?<=g_)[A-Z]{3}(?=_)")` and period `r stringr::str_extract(params$config, "(?<=_)[0-9]{4}_[0-9]{4}(?=\\.)")`
+# Mapping results summary for `r stringr::str_extract(params$config, "[A-Z]{3}")` and period `r stringr::str_extract(params$config, "[0-9]{4}_[0-9]{4}")`
```

Comment:
we should make this a function. not sure if it's needed for this pr, but at least we should open an issue.

Reply:
`extract_name_from_config_filename` and `extract_date_from_config_filename` seem reasonable.  We use this functionality elsewhere